### ISSUE TITLE
MiniLM 蒸馏自回归任务的问题

### ISSUE THREAD

**Issue by zhanghan1992** [microsoft/unilm#198]
请问下，对于 unilm 这种包含自回归任务的蒸馏：
1.Self-Attention Distribution Transfer中，  对齐的目标是 softmax(dot(query, key)) 还是  softmax(dot(query,key) + attention_mask) ？
2.Self-Attention Value-Relation Transfer中， dot(value, value)是否也需要加上 attention_mask？

**Comment by WenhuiWang0824** [comment]
Hi @zhanghan1992,  都加了attention_mask。

**Comment by zhanghan1992** [comment]
@WenhuiWang0824 感谢回答！再请教下，MiniLM的蒸馏过程中只用了 attention 和 value-relation 的loss, 二者的对齐能否让student model 在语言模型上收敛？ 有没有尝试过把语言模型的loss加上会不会在下游任务上取得更好的效果？
